src/commands/ext-dev-list.ts

```diff
@@ -0,0 +1,67 @@
+import * as clc from "cli-color";
+import Table = require("cli-table");
+import * as _ from "lodash";
+
+import { Command } from "../command";
+import { logPrefix } from "../extensions/extensionsHelper";
+import { FirebaseError } from "../error";
+import * as utils from "../utils";
+import * as extensionsUtils from "../extensions/utils";
+import { listExtensions } from "../extensions/extensionsApi";
+import * as logger from "../logger";
+import { requireAuth } from "../requireAuth";
+
+/**
+ * List all published extensions associated with this publisher ID
+ */
+export default new Command("ext:dev:list [publisherId]")
+  .description("list all published extensions associated with this publisher ID")
+  .before(requireAuth)
+  .action(async (publisherId: string, options: any) => {
+    if (!publisherId) {
+      throw new FirebaseError(
+        `You need to enter a publisher ID to view their published extensions. Expected Usage: ${clc.bold(
+          "ext:dev:list <publisher ID>"
+        )}`
+      );
+    }
```

Comment:
If you make publisherId required (with angle brackets), you _might_ not need this check.

If you _do_ need this check, make sure the parameters match and please print out the full command:

`firebase ext:dev:list <publisherId>`

Reply:
Done!